server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -866,4 +873,36 @@ export class AgenticChatController implements ChatHandlers {
     #debug(...messages: string[]) {
         this.#features.logging.debug(messages.join(' '))
     }
+
+    async #getRelevantDocuments(prompt: string): Promise<RelevantTextDocumentAddition[]> {
+        try {
+            this.#log('Querying relevant documents from local project index')
+            const localProjectContextController = LocalProjectContextController.getInstance()
+            const chunks = await localProjectContextController.queryVectorIndex({ query: prompt })
```

Comment:
We should explicitly check if indexing is enabled and port logic to prompt user to enable indexing. Otherwsie, customer will not know why his `@workspace` is not working. At least this is the current logic.